web/src/ChatManager.ts

```diff
@@ -171,11 +134,6 @@ class ChatManager {
   async fetchUpdate(): Promise<void> {
     try {
       const userMessages = await this.ledger.query(V4.Message);
```

Comment:
Maybe this should be just `messages`? `userMessages` reads like it's messages that is only belonging to the user which isn't accurate anymore.

Reply:
yeah ok I'll change

Reply:
done